_modules/dotnet-cli.js

```diff
@@ -45,9 +50,16 @@ const dotnetCli = {
         dir.pushd(dotnetPath.cliDir());
 
         // Dynamically find the latest dotnet core bin so that upgrades won't break this command
```

Comment:
Not sure this comment is still relevant here, I think this was probably ported over from the older version of this cli for the LMS project. If anything, it might make more sense in the `dotnet-path` module, but it might not be necessary.